queries.sql

```diff
@@ -1,3 +1,20 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE Name LIKE '%mon';
+
+SELECT * FROM animals WHERE
+EXTRACT( YEAR FROM date_of_birth ) BETWEEN 2016 AND 2019;
+
+SELECT * FROM animals WHERE
+neutered=true and escape_attempts<3;
+
+SELECT date_of_birth FROM animals WHERE
+name='Agumon' OR name='Pikachu';
+
+SELECT name, escape_attempts FROM animals WHERE
+weight_kg>10.5;
+
+SELECT * FROM animals WHERE name!='Gabumon';
+
+SELECT * FROM animals WHERE
+weight_kg>=10.4 and weight_kg<=17.3;
```

Comment:
- [ ] Kindly make sure you also add a query that can find all animals that are `neutered` as it misses in your list of queries. You can check your project's requirements to get all queries you should make.

Reply:
This query is already in the list at line `8` and `9`